.gitignore

```diff
@@ -57,6 +57,8 @@ distribute-*.tar.gz
 .tox
 .*.sw[op]
 *~
+#*
+.#*
```

Comment:
```suggestion
```
What files are these lines attempting to ignore?
The first one is a comment (`#*`) the second looks like it ignores (`.`) which is the current directory (already not included).

Reply:
These are some of the emacs save files that can be left if it crashes. Things like
#%2Aediff-merge%2A#hDh6cL#
and 
.#%2%2A#hDh6cL#

Reply:
Thanks. Would you move those to a section with a comment `# emacs crash files`?

The first example `#%2Aediff-merge%2A#hDh6cL#` isn't ignored with the current items but is ignored if you update `#*` to `\#*`. Would you also make that change?